### DISCUSSION TITLE
Is There a Way to Change Header Position From the Top?

### DISCUSSION THREAD

**Discussion by dyrector** [dolanmiu/docx#1415] [Q&A] [answered]
Is there a way to change the header position distance from the top? I need it to be 0.2". Actually, I also need the footer to be 0.2" from the bottom.

Here is the default I see in my DOCX document:

![image](https://user-images.githubusercontent.com/29614087/153013508-02780e87-3325-45f3-b8b9-618bb8ce4984.png)

**Comment by dolanmiu** [comment, marked as answer]
Yup, certainly

Check out this demo:

https://github.com/dolanmiu/docx/blob/master/demo/59-header-footer-margins.ts

**Reply by dyrector** [reply]
Thanks!
